sdk/python/kfp/dsl/_container_op.py

```diff
@@ -745,6 +746,17 @@ def add_volume(self, volume):
         self.volumes.append(volume)
         return self
 
+    def add_toleration(self, tolerations):
```

Comment:
You should probably specify the type of `tolerations`

Reply:
Right now it's not apparent to the user what kind of argument they need to pass here.

Reply:
added